### ISSUE TITLE
Unable to configure modsecurity as a reverse proxy

### ISSUE THREAD

**Issue by bguerout** [coreruleset/modsecurity-crs-docker#74]
Hello,

I'm trying to add modsecurity into a docker-compose stack.
The modsecurity (waf) is configured as a reverse proxy and should forward requests to a container named `server` 

```
waf:
  image: owasp/modsecurity-crs:3.3.2-nginx
  ports:
    - 80:80
  depends_on:
    - server
  environment:
    - SERVERNAME=localhost
    - PARANOIA=1
    - ANOMALY_INBOUND=5
    - ANOMALY_OUTBOUND=4
    - DNS_SERVER=127.0.0.11
    - PROXY=1
    - BACKEND=http://server/

server:...
```

Unfortunately even with DNS_SERVER provided, modsecurity is unable to resolve the upstream (ie. server). 

The modifications made by the entrypoint named `91-update-resolver.sh` seems to be erased by something (may be another entrypoint from `modsecurity-crs`). So the `resolver` directive is missing in `/etc/nginx/nginx.conf`

As a workaround, i've added a new entrypoint script 

```
  waf:
    image: owasp/modsecurity-crs:3.3.2-nginx
     ...
    volumes:
      - ./waf/99-update-resolver.sh:/docker-entrypoint.d/99-update-resolver.sh:ro
```

Entrypoint script:
```sh
#!/bin/sh
# vim:sw=2:ts=2:sts=2:et

set -eu

DNS_SERVER="${DNS_SERVER:-$(grep -i '^nameserver' /etc/resolv.conf | head -n1 | cut -d ' ' -f2)}"
RESOLVER_DIRECTIVE="resolver ${DNS_SERVER} ipv6=off valid=5s;"

sed -i.bak "/http {/a ${RESOLVER_DIRECTIVE}" /etc/nginx/nginx.conf
```

**Comment by bguerout** [comment]
After looking in the code, the template `nginx.conf.template` has been modified from the `modsecurity-docker` version and the following line does not longer exist

```
    resolver DNS_SERVER valid=5s;
```

`91-update-resolver.sh` entrypoint from `modsecurity-docker` is broken and it's no longer possible to configure the resolver `directive` with `DNS_SERVER` variable.

**Comment by theseion** [comment]
Fixed by #79. @bguerout can you verify?
